stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/channel/ChannelController.kt

```diff
@@ -278,33 +279,13 @@ public class ChannelController internal constructor(
         }
 
```

Comment:
Could we also document the functions here that are being touched but don't have KDocs?